mon-pix/app/pods/components/module/qcu/template.hbs

```diff
@@ -30,7 +30,13 @@
   {{/if}}
 
   {{#unless this.qcu.lastCorrection}}
-    <PixButton @backgroundColor="green" @shape="rounded" @type="submit" class="element-qcu__verify_button">
+    <PixButton
+      @backgroundColor="green"
+      @shape="rounded"
+      @type="submit"
+      class="element-qcu__verify_button"
+      @triggerAction={{this.submitAnswer}}
+    >
```

Comment:
du coup ne doit-on pas enlever le `{{on "submit" this.submitAnswer}}` sur l'élément `form` ?

Reply:
En ne le précisant pas, le `button` prend la valeur par défaut `type="button"`. Je supposait que sémantiquement le type `"submit"` était nécessaire pour pouvoir soumettre à l'appui de entrée mais après test ça a l'air de marcher sans...

https://codepen.io/yannbertrand/pen/LYajeGP

Reply:
Du coup ça fait doublon non ? 🤔

Reply:
Je pense que c'est un peu trompeur de ne pas préciser le type au niveau du `button`, même si dans notre implem de `PixButton` il ne semble pas nécessaire. Ça appuie sémantiquement ce à quoi sert ce bouton. Ça prendra plus de sens quand on ajoutera d'autres boutons à côté.

Reply:
J'ai supprimé le `{{on "submit" this.submitAnswer}}`, ça a l'air de fonctionner pareil